lib/api/src/grpc/conversions.rs

```diff
@@ -904,22 +904,46 @@ impl TryFrom<GeoPolygon> for segment::types::GeoPolygon {
     type Error = Status;
 
     fn try_from(value: GeoPolygon) -> Result<Self, Self::Error> {
-        if value.points.is_empty() {
+        if value.lines.is_empty() {
             return Err(Status::invalid_argument("Empty GeoPolygon"));
         }
 
-        let points: Vec<segment::types::GeoPoint> =
-            value.points.into_iter().map(Into::into).collect();
+        let exterior_lines: Vec<segment::types::GeoPoint> = match value.lines.get(0) {
+            Some(line) => line.points.iter().cloned().map(Into::into).collect(),
+            None => return Err(Status::invalid_argument("Missing exterior boundary")),
+        };
+
+        let interior_lines: Vec<Vec<segment::types::GeoPoint>> = value.lines[1..]
+            .iter()
+            .map(|line| line.points.iter().cloned().map(Into::into).collect())
+            .collect();
 
-        Ok(Self { points })
+        Ok(Self {
```

Comment:
But the gRPC follows the standard strictly regarding exterior rings.

I wonder if we should have both REST & gRPC do the same thing.

Reply:
Changed RESTand gRPC to the same.